cypress-tests/integration-tests/step-definitions/search.ts

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable func-names */
```

Comment:
Is this comment needed?

Reply:
Good point. It was there from the step definition I copied from, and was preventing linting errors due to unnamed functions. However, there's only one unnamed function in this file, and it doesn't need to be.